event.go

```diff
@@ -828,3 +830,21 @@ func (e *Event) MACAddr(key string, ha net.HardwareAddr) *Event {
 	e.buf = enc.AppendMACAddr(enc.AppendKey(e.buf, key), ha)
 	return e
 }
+
+// GetFields returns the JSON decoded fields of the event. Please note that the fields might not be mapped to their original type as JSON decoder maps numbers to float64, etc.
+func (e *Event) GetFields() (map[string]interface{}, error) {
```

Comment:
Is it really about fields if you are getting the values with them?

Maybe GetMetadata or GetJSON would be a better?

Reply:
`GetMetadata` works I think as we're not getting a JSON (we're parsing a JSON).